cost/azure/databricks/rightsize_compute/azure_databricks_rightsize_compute.pt

```diff
@@ -0,0 +1,1478 @@
+        if (typeof vm["monthly_cost_estimate"] == "number") {
+          vm["savings"] = vm["monthly_cost_estimate"] // Potential Savings from delete is the estimated monthly cost of the resource
+        }
+        vm["memoryThreshold"] = param_stats_idle_threshold_mem_value
+        vm["cpuThreshold"] = param_stats_idle_threshold_cpu_value
+        idle_list.push(vm)
+      } else if (is_underutil) { // If not idle, check see if the resource is below the lower "idle" threshold
+        // Get the downsize instance type
+        if (ds_azure_instance_size_map[vm['resourceType']]) {
+          vm["newResourceType"] = ds_azure_instance_size_map[vm['resourceType']]['down']
+        }
+        vm["recommendationType"] = "Downsize"
+        vm["recommendationDetails"] = [
+          "Resize Azure virtual machine ", vm["resourceId"], " ",
+          "in Azure Subscription ", vm["subscriptionName"], " ",
+          "(", vm["subscriptionId"], ") ",
+          "from ", vm["resourceType"], " ",
+          "to ", vm["newResourceType"]
+        ].join("")
+        if (typeof vm["monthly_cost_estimate"] == "number") {
+          vm["savings"] = vm["monthly_cost_estimate"] / 2 // Potential Savings from downsize is generally half the estimated monthly cost of current resource type
+        }
+        vm["memoryThreshold"] = param_stats_underutil_threshold_mem_value
+        vm["cpuThreshold"] = param_stats_underutil_threshold_cpu_value
+        underutil_list.push(vm)
+      }
+    }
+  })
+  result = {
+    "underutil_list": underutil_list,
+    "idle_list": idle_list
+  }
+  EOS
+end
+
+datasource "ds_underutilized_instances" do
+  run_script $js_underutilized_instances, $ds_idle_and_underutil_instances
+end
+
+script "js_underutilized_instances", type: "javascript" do
+  parameters "ds_idle_and_underutil_instances"
+  result "result"
+  code <<-EOS
+  if (typeof ds_idle_and_underutil_instances["underutil_list"] != "undefined" && ds_idle_and_underutil_instances["underutil_list"].length > 0) {
+    result = ds_idle_and_underutil_instances["underutil_list"]
+  } else {
+    result = []
+  }
+EOS
+end
+
+datasource "ds_idle_instances" do
+  run_script $js_idle_instances, $ds_idle_and_underutil_instances
+end
+
+script "js_idle_instances", type: "javascript" do
+  parameters "ds_idle_and_underutil_instances"
+  result "result"
+  code <<-EOS
+  if (typeof ds_idle_and_underutil_instances["idle_list"] != "undefined" && ds_idle_and_underutil_instances["idle_list"].length > 0) {
+    result = ds_idle_and_underutil_instances["idle_list"]
+  } else {
+    result = []
+  }
+EOS
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "pol_databricks_optimizations" do
+  validate $ds_idle_instances do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Databricks compute instances that are Idle"
```

Comment:
Should update this to use the name of the applied policy instead, similar to Azure Rightsize Compute.

Reply:
Fixed in 2855dd50a0d92a0330e8c516580faafca532adf6